app/build.gradle.kts

```diff
@@ -44,6 +45,10 @@ android {
 
 	composeOptions {
 		kotlinCompilerExtensionVersion = "1.5.7"
+		warnln(
+			"The Compose Compiler 1.5.7 is not compatible with the newest version of Kotlin 1.9.22. " +
```

Comment:
@MarekMacko The idea was to warn another dev to not update Kotlin to 1.9.22 as it would crash the app. But now I think it should be next to the place where the Kotlin version is listed. Just to save the time.

Reply:
there is a renovate bot cofigured. In this it won't even build the project so you're not able to merge it.

Reply:
@MarekMacko Yep, but it will take some time for somebody, most likely me, to figure it out again, and again when renovate will try to update the Kotlin. Then this comment can be useful. At this moment I am not sure if there was a clear the error of this, or I had to google it. But I guess the second.

Reply:
I think everyone using Compose should remember that Compose needs to be updated to use a newer Kotlin version, so this warning is redundant to me. At first glance maybe it looks like it could be helpful but IMO it could also be misleading because the warning will need to be updated with the newer Kotlin version. Who will remember that? :D